contracts/ChainlinkRegistry/ChainlinkRegistry.sol

```diff
@@ -73,3 +81,7 @@ contract ChainlinkRegistry is Governable, CollectableDust, IChainlinkRegistry {
     return keccak256(abi.encodePacked(_base, _quote));
   }
 }
+
+interface IAggregatorProxy is AggregatorV2V3Interface {
+  function aggregator() external view returns (address);
+}
```

Comment:
This shouldn't be here?

Reply:
I need it only on the `_isProxy` function, because Chainlink doesn't expose an interface for their proxies. Since this only meant to be used internally, I though it didn't make sense to move it to the public interfaces folder